# Generated for NaoQi method ALFrameManager.onPythonError
# Transmit Python error to Choregraphe or logger. Should not be called manually.

# Name of the box raising the error
string boxname
# Name of the method
string methodname
# Error string
string error
---


